### PR TITLE
NEW RemoteUtilities.Host v7.1.7.0

### PR DESCRIPTION
**Pull request by mluepkes**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90659)

### CODE REVIEW

**Comment by mluepkes** [comment]
Any advice on what the problem exactly is? I can't find any error descriptions in the pipeline and the SandboxTest.ps1 runs just fine.

**Comment by stephengillie** [comment]
In my sandbox it also installs fine. In my VM it errors that it's missing MSVCR120.dll. 
image

I've verified this is a VCRedist package, and while I haven't found the specific package needed here, I've tested with Microsoft.VCRedist.2015+.x64 and still received the error, so I'm suspecting another package is needed as a dependency here.

**Comment by mdanish-kh** [comment]
@msftbot Dependencies

**Comment by stephengillie** [comment]
@mluepkes were you able to find the dependency?

**Comment by mluepkes** [comment]
I just got around to trying another install of the manifest in my sandbox. I'm still failing to reproduce the error (Win 11 22H2 Build 22621.963). Installation runs through w/o any errors thrown. I can start and stop the service, access and use the settings panel without a problem. Without being able to repro there's not much I can do.

File: manifests/r/RemoteUtilities/Host/7.1.7.0/RemoteUtilities.Host.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RemoteUtilities.Host
+PackageVersion: 7.1.7.0
+Installers:
```
Comment (stephengillie) [inline manifests/r/RemoteUtilities/Host/7.1.7.0/RemoteUtilities.Host.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2012.x64
Installers:
```

Comment (stephengillie) [reply manifests/r/RemoteUtilities/Host/7.1.7.0/RemoteUtilities.Host.installer.yaml:9]: Research yielded a mapping of DLL name to VCRedist package version: 
MSVCR71.dll: 2005?
MSVCR08.dll: 2008
MSVCR09.dll: 2010
MSVCR120.dl:| 2012
MSVCR130.dll: 2013
MSVCR140.dll: 2015+

Based on this, the Microsoft.VCRedist.2012.x64 is necessary. 
- Testing in my VM provided an odd state: after installing dependency and then package, the executables still error that the DLL is missing. Running the same `winget install` command and running the installer again were necessary to clear the error - only after this 2nd installer run did the executables launch without error. The VCRedist dependency did not reinstall on the 2nd run.

**Comment by stephengillie** [comment]
I've found that a 2nd run of the install process has been necessary to get the package to recognize that VCRedist is installed. Winget installs dependencies before the main package, but this doesn't remove the MSVCR120.dll error above. Only after running `Winget install` twice are the applications able to locate the DLL. 
1. The first run installs both the VCRedist dependency and the package.
2. The second run installs only the package. 

I think the automated validation pipeline is blocked from downloading the file. In the validation logs, I see this:
```
##[error] Installer https://www.remoteutilities.com/download/host-7.1.7.0.msi
Scan request id: '858addad-b95a-4d95-82f8-44c2976116db'
Error information: BlockingDetectionFound
##[error] Result: Failed
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**